_includes/scripts.html

```diff
@@ -39,4 +36,64 @@
 </script>
 <!-- Intercom -->
 <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/rzcjacgd';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+<script>
+  (function ($) {
```

Comment:
Why move this from `main.js` and embed in HTML? Just to ensure it loads after Intercom code? If so, probably cleaner to define a function in `main.js` that has all the code below and to just call that function in the `script` tag here.

Reply:
Moved the cookie code to its own file so it can load asynchronously and won't affect load time.

Reply:
Oh, are you saying that the `main.js` file may take a while to load, whereas the `cookie.js` file being shorter will load right away? Either way, we're already loading a bunch of separate javascript files (albeit mostly from third parties) so I think this is fine for now.